packages/@react-aria/tooltip/docs/useTooltipTrigger.mdx

```diff
@@ -77,7 +77,7 @@ This example implements a Tooltip that renders in an absolute position next to i
 
 Two instances of the example are rendered to demonstrate the behavior of the delay on hover. If you hover over the first button, the tooltip will be shown after a delay. Hovering over the second button shows the tooltip immediately. If you wait for a delay before hovering another element, the delay restarts.
 
-```tsx example
+```tsx example export=true
```

Comment:
should we merge useTooltipTrigger with useTooltip like you did for Menu and MenuTrigger?

Reply:
I only see https://react-spectrum.adobe.com/react-aria/useTooltipTrigger.html, there is no useTooltip page. What do you mean?

Reply:
I have no idea what I thought I saw, you are correct